### DISCUSSION TITLE
Regex in Babashka

### DISCUSSION THREAD

**Discussion by diegonayalazo** [babashka/babashka#1167] [Q&A]
Hi borkdude, thanks again for Babashka. I don't know if I am doing something wrong but 

(str/split "Clojure is awesome!" #" ")

This is not working in BB. The following error ensues:

Exception in thread "main" java.lang.RuntimeException: No dispatch macro for: "

Is the Clojure Regex syntax supported? 

Thanks for your help.

**Comment by borkdude** [comment]
@diegonayalazo Are you doing this in a `bb.edn` file? This syntax isn't support in .edn files, that is not a babashka, but a Clojure restriction. You can use `(re-pattern " ")` instead.

**Reply by diegonayalazo** [reply]
Thanks for the fast response. Yes Indeed, this is an bb.edn file. I will research the use of re-pattern instead. Greatly appreciated.

**Reply by borkdude** [reply]
So, in Clojure `#"foo"` is the same as `(re-pattern "foo")`. But this syntax is only allowed in `.clj*` files.

**Comment by borkdude** [comment]
@diegonayalazo Feel free to ask as many questions as you like. 

Next time, you can open a question directly in the Discussions part of this repo: https://github.com/babashka/babashka/discussions/categories/q-a

**Reply by diegonayalazo** [reply]
Bookmarked!
